core/src/banking_stage/transaction_scheduler/prio_graph_scheduler.rs

```diff
@@ -55,6 +58,7 @@ impl PrioGraphScheduler {
     pub(crate) fn schedule(
         &mut self,
         container: &mut TransactionStateContainer,
+        filter: impl Fn(&SanitizedTransaction) -> bool,
```

Comment:
Design choice here for using a generic `filter` here.
IMO, this greatly simplifies much of the testing, since we don't need a bank there.

Reply:
Ah no, I strongly prefer a fn here compared to a trait. Trait doesn't let me use a closure if I wanted